index.html

```diff
@@ -0,0 +1,157 @@
+    </h1>
+    <p class="intro">
+      <strong>QWC 2022 </strong> , invites you for an opening ceremony to celebrate football on and off the pitch at the
+      central fan destination in Doha-Qatar where history will be made on 16th NOvember 2022. <strong>32</strong>
+      countries will compete for the trophy.
+    </p>
+    <h3 class="event-date">2022.11.16(mon) ~ 19(wed)</h3>
+    <h4 class="event-location">@ Al Bayt Stadium | Doha |</h4>
+  </header>
+  <main>
+    <section class="program-section" id="program">
+      <h2 class="sec-title">
+        main program
+        <span class="indicator"></span>
+      </h2>
+      <div class="cards">
+        <div class="card">
+          <img src="icons/microphone.png" alt="speaker-icon" />
+          <h3 id="artiste">Artistes Performing</h3>
+          <p id="musicians">
+            Musicians from different continents will entertain 
+          </p>
+        </div>
+        <div class="card">
+          <img  src="icons/speech.png" alt="speech" /><br>
+          <h3 id="speech" >Speech</h3>
+          <p id="talk">
+            Speech from different football legends of world cup, and from the main speaker(Qatar president)
+          </p>
+        </div>
+        <div class="card">
+          <img src="icons/trophy.png" alt="trophy" />
+          <h3 id="trophy">WorldCup Trophy</h3>
+          <p>
+            The trophy that will be competed for by 32 countries will be unveiled by Qatar President.
+          </p>
+        </div>
+        <div class="card">
+          <img src="icons/sponsors.png" alt="Sponsors" />
+          <h3 id="sponsor">Sponsors</h3>
+          <p id="sponsors">
+            Sponsors for this edition of worldcup will be unveiled by the FIFA president.
+          </p>
+        </div>
+        <div class="card">
+          <img src="icons/tv.png" alt="tv" />
+          <h3 id="tvs">TV broadcast</h3>
+          <p id="tv">
+            All televisions that will broadcast all matches will be unveiled by the FIFA president.
+          </p>
+        </div>
+      </div>
+      <button type="button" class="primary-btn">
+        Don't miss the World's <br> Entertaining Football
+      </button>
+    </section>
+    <section class="speakers-section" id="speakers">
+      <h2 class="dummy-heading">
+        This heading is used for linter pass only
+      </h2>
+      <!-- Generated dynamically -->
+      </section>
+      <section class="partner-section" id="partner">
+        <h2 class="sec-title">
+          partners
+          <span class="indicator"></span>
+        </h2>
+        <div class="partners-logo-container">
+          <img src="images/Partners/cocacola.png" alt="cocacola" />
+          <img src="images/Partners/adidas.png" alt="adidas" />
+          <img src="images/Partners/hp.png" alt="hp-logo" />
+          <img src="images/Partners/MTN-Logo.png" alt="MTN-Logo" />
+          <img src="images/Partners/standard-chartered.png" alt="standard-chartered-logo" />
```

Comment:
Kindly reduce the size of the partners' icons to match the given design. They're currently larger.

Your design:
![image](https://user-images.githubusercontent.com/20554583/190397966-2c0cf02f-3828-451e-8d7a-d4c69f21b15e.png)

Cindy's design:
![image](https://user-images.githubusercontent.com/20554583/190398218-548a90c0-3fee-42ea-9450-8913bc23fea4.png)